.github/workflows/CI.yaml

```diff
@@ -70,10 +70,35 @@ jobs:
         pytest -v --cov=membrane_curvature --cov-report=xml --color=yes membrane_curvature/tests/
 
     - name: CodeCov
-      uses: codecov/codecov-action@v1
+      uses: codecov/codecov-action@v2
       with:
         file: ./coverage.xml
         flags: unittests
         name: codecov-${{ matrix.os }}-py${{ matrix.python-version }}
         fail_ci_if_error: True
 
+  latest_python:
```

Comment:
You're really doing a "latest everything" here (since it's also the only place you're checking against MDA develop [as far as I'm aware?]). So a name like "upstream_develop" or something might be more appropriate?

Reply:
Yes, that's a good point. Updated.